adapters/adapter.go

```diff
@@ -119,7 +119,8 @@ func For(task models.TaskSpec, store *store.Store) (*PipelineAdapter, error) {
 		if err != nil {
 			return nil, fmt.Errorf("%s is not a supported adapter type", task.Type)
 		}
-		b := Bridge{BridgeType: bt, Params: &task.Params}
+		pcopy := task.Params
```

Comment:
Are both of these copies needed? (TaskSpec and Params?)

Reply:
No, I've removed it.